libethereum/EthereumCapability.cpp

```diff
@@ -31,17 +19,19 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
-static unsigned const c_maxSendTransactions = 256;
-static unsigned const c_maxHeadersToSend = 1024;
-static unsigned const c_maxIncomingNewHashes = 1024;
-static int const c_backroundWorkPeriodMs = 1000;
-static int const c_minBlockBroadcastPeers = 4;
-
 char const* const EthereumCapability::s_stateNames[static_cast<int>(SyncState::Size)] = {
     "NotSynced", "Idle", "Waiting", "Blocks", "State"};
 
+chrono::milliseconds constexpr EthereumCapability::s_backgroundWorkInterval;
+
 namespace
 {
+static constexpr unsigned c_maxSendTransactions = 256;
```

Comment:
Having it in the unnamed namespace and making it `static` has the exact same effect, so having both is redundant. If you move it, remove `static`

Reply:
Ah you're right, objects defined in namespace scope have static storage duration. Thanks for calling this out!